src/components/vm/consoles/consoles.jsx

```diff
@@ -34,11 +43,70 @@ import './consoles.css';
 
 const _ = cockpit.gettext;
 
-const VmNotRunning = () => {
+const ConsoleEmptyState = ({ vm }) => {
+    const Dialogs = useDialogs();
+    const serials = vm.displays && vm.displays.filter(display => display.type == 'pty');
+    const inactive_vnc = vm.inactiveXML.displays && vm.inactiveXML.displays.find(display => display.type == 'vnc');
+
+    function add_vnc() {
+        Dialogs.show(<AddEditVNCModal idPrefix="add-vnc" vm={vm} consoleDetail={null} />);
+    }
+
+    function edit_vnc() {
+        Dialogs.show(<AddEditVNCModal idPrefix="edit-vnc" vm={vm} consoleDetail={inactive_vnc} />);
+    }
+
+    let icon = null;
+    let top_message = null;
+    let bot_message = _("Graphical console support not enabled");
```

Comment:
There is still this language impedance mismatch between "Graphical console support not enabled" and "Add VNC". As there's also SPICE, let's disambiguate by either "VNC console not enabled" (the "support" doesn't add anything here"), or perhaps "Graphical VNC console not enabled".

Reply:
@garrett?